File: internal/model/suite.go
package model

type Suite struct {
	BaseModel
	SuiteCode       string `gorm:"column:suite_code;size:60" json:"suiteCode"`
	SuiteName       string `gorm:"column:suite_name;size:30" json:"suiteName"`
	SuiteClassifyID *int64 `gorm:"column:suite_classify_id" json:"suiteClassifyId,omitempty"`
	SuiteImage      string `gorm:"column:suite_image;type:text" json:"suiteImage,omitempty"`
	SuiteVersion    string `gorm:"column:suite_version;size:10" json:"suiteVersion,omitempty"`
	SuiteDesc       string `gorm:"column:suite_desc;size:140" json:"suiteDesc,omitempty"`
	SuiteHelpDocJSON string `gorm:"column:suite_help_doc_json;size:300" json:"suiteHelpDocJson,omitempty"`
	SuiteFlag       *int   `gorm:"column:suite_flag" json:"suiteFlag,omitempty"`
}

func (Suite) TableName() string { return "t_suite" }

File: commom/src/main/java/com/javaoffers/nexus/common/model/Suite.java
package com.javaoffers.nexus.common.model;

import com.javaoffers.brief.modelhelper.anno.BaseModel;
import com.javaoffers.brief.modelhelper.anno.BaseUnique;
import com.javaoffers.brief.modelhelper.anno.ColName;
import com.javaoffers.brief.modelhelper.anno.derive.flag.IsDel;
import lombok.Data;

import java.util.Date;

@BaseModel("t_suite")
@Data
public class Suite {
    @BaseUnique
    private Long id;
    private IsDel deleted;
    private Date createdAt;
    private Long createdBy;
    private Date updatedAt;
    private Long updatedBy;

    @ColName(value = "id", excludeColAll = true)
    private Long countId;

    private String suiteCode;
    private String suiteName;
    private Long suiteClassifyId;
    private String suiteImage;
    private String suiteVersion;
    private String suiteDesc;
    private String suiteHelpDocJson;
    private Integer suiteFlag;
}
